components/Dropdown/Dropdown.js

```diff
@@ -1,28 +1,21 @@
-class Dropdown {
+class Menu {
   constructor(element) {
-    
-    // Assign this.element to the dropdown element
-    this.element;
-    
-    // Get the element with the ".dropdown-button" class found in the dropdown element (look at the HTML for context)
-    this.button = this.element.querySelector();
-    
-    // assign the reference to the ".dropdown-content" class found in the dropdown element
-    this.content;
-    
-    // Add a click handler to the button reference and call the toggleContent method.
-    this.button.addEventListener('click', () => {
-
-    })
+    this.Menu = element;
```

Comment:
so long as you're referencing the `menu` object being passed in and not the `Menu` class constructor, I'd stick to lowercase here to avoid confusion